icu4c/source/data/unidata/generate.sh

```diff
@@ -30,6 +30,9 @@ rm $ICU4C_DATA_IN/coll/*.icu
 # Exit this shell script when a command fails.
 set -e
 
+# Workaround for obscure bug with the Bazel cache (ICU-22193).
```

Comment:
Would you mind moving this comment / commit message / PR title to ICU-22233?

Reply:
Done.